README.md

```diff
@@ -88,29 +92,22 @@ This section provides instructions to develop custom measurement services in Pyt
 
 ### Installation
 
-Make sure the system has the recommended Python version is installed. Install Measurement Plug-In SDK for Python using [pip](https://pip.pypa.io/).
+Make sure the system has the recommended Python version installed. Install Measurement Plug-In SDK for Python using [pip](https://pip.pypa.io/).
 
 ``` cmd
 REM Activate the required virtual environment if any.
-pip install ni-measurement-plugin-sdk-service
+pip install ni-measurement-plugin-sdk
 ```
 
 Check if you have installed the expected version of Measurement Plug-In SDK for Python installed by running the below command:
 
 ```cmd
-pip show ni-measurement-plugin-sdk-service
+pip show ni-measurement-plugin-sdk
 ```
 
 ### Developing a minimal Python measurement
 
-1. Install the `ni-measurement-plugin-sdk-generator` package.
-
-``` cmd
-REM Activate the required virtual environment if any.
-pip install ni-measurement-plugin-sdk-generator
-```
-
-2. Run the `ni-measurement-plugin-sdk-generator` tool. Use command line arguments to specify the `display-name` and optionally the `version`, `measurement-type`, and `product-type`.
+1. Run the `ni-measurement-plugin-sdk-generator` tool. Use command line arguments to specify the `display-name` and optionally the `version`, `measurement-type`, and `product-type`.
 
     1. Running `ni-measurement-plugin-sdk-generator` without optional arguments:
```

Comment:
@bkeryan @dixonjoel 
According to the command [here](https://github.com/ni/measurement-plugin-python/blob/main/packages/generator/pyproject.toml#L45), could you please confirm if we need to change `ni-measurement-plugin-sdk-generator` to `ni-measurement-plugin-generator`?

Reply:
yes

Reply:
Thanks for the confirmation. I've changed the tool name in all applicable readme files.